---
int32 longitude
string positionLongitude
int32 latitude
string positionLatitude
